lib/lightspeed/request.rb

```diff
@@ -13,33 +15,32 @@ def self.verbose?
       !! @verbose
     end
 
-    def self.base_url
-      "https://api.merchantos.com/API"
+    def self.base_host
+      "api.merchantos.com"
+    end
+
+    def self.base_path
+      "/API"
     end
 
     def initialize(client, method:, path:, params: nil, body: nil)
+      @method = method
```

Comment:
do you mean the public accessors for `raw_request`, `bucket_level` and `bucket_max` or these "private" instance vars? If you are referring to the public accessors, some guidance on designing around them would help. If you mean the instance vars, I'm happy to go along with your preference of not using them, which is the way I wrote it up initially and refactored to tighten up the initialize method.